src/Provider/ShopProvider.php

```diff
@@ -296,4 +312,42 @@ public function getUrl($shopId)
 
         return new ShopUrl($backOfficeUrl, $frontendUrl, $shopId);
     }
+
+    /**
+     * @return array
+     *
+     * @throws \PrestaShopException
+     * @throws \Exception
```

Comment:
Not sure it's very usefull to mention thoses exceptions as we probably never want to catch them ?